src/rp2_common/hardware_clocks/include/hardware/clocks.h

```diff
@@ -318,6 +318,31 @@ typedef clock_num_t clock_handle_t;
  */
 bool clock_configure(clock_handle_t clock, uint32_t src, uint32_t auxsrc, uint32_t src_freq, uint32_t freq);
 
+/*! \brief Configure the specified clock to +/- 1MHz
```

Comment:
Actually could we clarify what this means per our discussion

Reply:
I think this is correct

```suggestion
/*! \brief Configure the specified clock with 1MHz accuracy
```